test/test-bopomofo.c

```diff
@@ -1690,6 +1691,50 @@ void test_KB()
     test_KB_MPS2();
 }
 
+void test_chewing_phone_to_bopomofo()
+{
+    char *u8phone;
+    char *rt;
+    int  len;
+    uint16_t phone;
+    uint16_t expect;
+
+    start_testcase(NULL, fd);
+
+    u8phone = "\xE3\x84\x86\xE3\x84\xA3" /* ㄆㄣ */ ;
+    phone = UintFromPhone(u8phone);
+    expect = 1104;
+    ok(phone == expect, "UintFromPhone `%s' shall be `%d', got `%d'", u8phone, expect, phone);
+
+    len = chewing_phone_to_bopomofo(expect,NULL,0);
+    rt = calloc(sizeof(char), len);
+    chewing_phone_to_bopomofo(expect, rt, len );
+    ok(strcmp(rt, u8phone) == 0, "PhoneFromUint d%d' shall be `%s', got `%s'", expect, u8phone, rt);
+    free(rt);
+
+    u8phone = "\xE3\x84\x8A\xE3\x84\xA7\xE3\x84\xA2" /* ㄊㄧㄢ */ ;
+    phone = UintFromPhone(u8phone);
+    expect = 3272;
+    ok(phone == expect, "UintFromPhone `%s' shall be `%d', got `%d'", u8phone, expect, phone);
+
+    len = chewing_phone_to_bopomofo(expect,NULL,0);
+    rt = calloc(sizeof(char), len);
+    chewing_phone_to_bopomofo(expect, rt, len );
+    ok(strcmp(rt, u8phone) == 0, "PhoneFromUint d%d' shall be `%s', got `%s'", expect, u8phone, rt);
+    free(rt);
+
+    u8phone = "\xE3\x84\x92\xE3\x84\xA7\xE3\x84\x9A\xCB\x8B" /* ㄒㄧㄚˋ */ ;
+    phone = UintFromPhone(u8phone);
+    expect = 7308;
+    ok(phone == expect, "UintFromPhone `%s' shall be `%d', got `%d'", u8phone, expect, phone);
+
+    len = chewing_phone_to_bopomofo(expect,NULL,0);
```

Comment:
should be `(expect, NULL, 0)`. The spaces are required to satisfy the indentation.